addons/crm/models/crm_lead.py

```diff
@@ -85,17 +86,9 @@
 PLS_UPDATE_BATCH_STEP = 5000
 
 
-class CrmLead(models.Model):
+class CrmLead(models.Model, mail.MailThreadCc, mail.MailThreadBlacklist, phone_validation.MailThreadPhone, mail.MailActivityMixin, calendar.MailActivityMixin, utm.UtmMixin, base.FormatAddressMixin, mail.MailTrackingDurationMixin):
```

Comment:
That does not makes sense: why changing something in calendar should change crm ? This globally break our mixin usage.

Reply:
I take the first level dependencies used in the `_inherit`. Here `MailActivityMixin` has a modification into `calendar` and `calendar` is a `crm` dependency. This is to have the autocompletion of the inheritance used.
I can perhaps limit this when we do an inherit and not when we declare a new model.

manifest: 'base_setup', 'sales_team', 'mail', '`calendar`', 'resource', 'utm', 'web_tour', 'contacts', 'digest', 'phone_validation'

Reply:
But we should not have to care what calendar does. If someone adds an override of a mixin in a module, we are not going to change the declaration of all depending classes, that's nonsense.

If that's only for autocompletion and has no other usage, please remove that from all SM addons.